ui/src/clockface/components/empty_state/EmptyState.tsx

```diff
@@ -16,6 +16,7 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 
 interface PassedProps {
   children: JSX.Element | JSX.Element[]
+  className?: string
```

Comment:
I think most of the other clockface components are using `customClass` for a prop like this